tests/e2e/test_rosacli_kubelet_config.go

```diff
@@ -54,9 +54,9 @@ var _ = Describe("Edit kubeletconfig",
 					Expect(output.String()).To(ContainSubstring("Hosted Control Plane clusters do not support custom KubeletConfig configuration."))
```

Comment:
From logs of job run :- https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_rosa/2043/pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-hcp/1790624905453113344
it seems that the `Hosted Control Plane clusters do **support** custom KubeletConfig configuration.`
i think we need to drop the validation blocks (if isHosted)  for HCP. @xueli181114 WDYT?